src/grids.jl

```diff
@@ -99,6 +99,26 @@ end
 # IMPLEMENTATIONS
 # ----------------
 
+geotiff(::Type{NAD27}, ::Type{NAD83}) = "us_noaa_conus.tif"
+
+geotiff(::Type{NAD27}, ::Type{NAD83CSRS{2}}) = "ca_nrc_NA27SCRS.tif"
+
+geotiff(::Type{NAD27}, ::Type{NAD83CSRS{3}}) = "ca_nrc_TO27CSv1.tif"
+
+# TODO: unsupported GeoTIFF format
+# geotiff(::Type{NAD27}, ::Type{NAD83CSRS{4}}) = "ca_nrc_BC_27_05.tif"
+
+# geotiff(::Type{NAD83}, ::Type{NAD83CSRS{2}}) = "ca_nrc_NA83SCRS.tif"
+
+# geotiff(::Type{NAD83}, ::Type{NAD83CSRS{3}}) = "ca_nrc_ON83CSv1.tif"
+
+# geotiff(::Type{NAD83}, ::Type{NAD83CSRS{4}}) = "ca_nrc_BC_93_05.tif"
+
+# TODO: grid files not found
+# geotiff(::Type{NAD83}, ::Type{NAD83CSRS{6}}) = ""
+
+# geotiff(::Type{NAD83}, ::Type{NAD83CSRS{7}}) = ""
```

Comment:
If it is not found because it is not available in EPSG, then we should just erase these lines from the file. If it is not found in the CDN, then we need to leave the lines here to check in the future.

Reply:
Not found in PROJ CDN